db/migrate/20190408125610_create_past_events.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePastEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :past_events do |t|
+      t.string :title,      null: false, default: ""
+      t.string :subtitle
+      t.text :description
+      t.integer :year,      null: false, default: ""
+      t.text :logo
+
+      t.timestamps
+    end
+    add_index :past_events, :title,           unique: true
```

Comment:
hmm title doesn't need to be unique? probably can add another field "slug" for easier URL referencing?

Reply:
I will remove the unique constraint when I am redesigning the database.